rudder-webapp/SOURCES/rudder-upgrade

```diff
@@ -397,6 +398,59 @@ rudder.cfengine.reload.server.command=/opt/rudder/bin/rudder-reload-cf-serverd"
   check_and_add_config_property rudder.jdbc.maxPoolSize "
 # Size of the PostgreSQL connection pool
 rudder.jdbc.maxPoolSize=25"
+
+
+  # - 3.2.0 : Add properties to configure authentication plugins and master admin
+  # note: we can't use rudder.auth.admin.login (or .password) because it's commented
+  # out by default, so it would defeat the regex. 
+  # We also need to comment out rudder.auth.ldap.enable and use its value to decide 
+  # if we should use "ldap" of "file" for rudder.auth.type 
+
+  STEP="Add properties to configure authentication plugins and master admin"
+  if grep -iEq "^rudder.auth.ldap.enable\s*=\s*false" /opt/rudder/etc/rudder-web.properties; then
```

Comment:
the line was mandatory, defaulting to rudder.auth.ldap.enable=false (so to the new "file" authentication provider).